kubernetes/deployment-production.tmpl

```diff
@@ -61,7 +61,7 @@ spec:
             - containerPort: 3000
           env:
             - name: PROJECT_ASSET_PREFIX
-              value: https://fe-project.zooniverse.org
+              value: https://fe-project.zooniverse.org/projects
```

Comment:
This can be changed to `https://www.zooniverse.org/projects`, once routing is set up.